internal/makefile/makefile.go

```diff
@@ -83,7 +83,15 @@ endif
 	test := category{name: "test"}
 
 	test.addDefinition(`# which packages to test with "go test"`)
-	test.addDefinition(`GO_TESTPKGS := $(shell go list -f '{{if or .TestGoFiles .XTestGoFiles}}{{.ImportPath}}{{end}}' ./...)`)
+	testPkgGreps := ""
+	if cfg.Test.Only != "" {
+		testPkgGreps += fmt.Sprintf(" | command grep -E '%s'", cfg.Test.Only)
```

Comment:
Is there a specific reason for why `command grep` is used instead of `grep`? Same three lines below.

Reply:
`command` was already there from before and was carried over during copy-paste. It is not needed and has been removed.

Reply:
In that case, feel free to revert my commit, if needed.

Reply:
I think that's a bug in Sandro's shell setup. You should not alias commands to other incompatible commands and expect things to work. I believe there is a way to set up aliases only for interactive use, so that would be the proper fix IMO.